ubcpi/ubcpi.py

```diff
@@ -442,10 +445,17 @@ def student_view(self, context=None):
         random.seed(student_item['student_id'])
 
         answers = self.get_answers_for_student()
-        html = ""
-        html += self.resource_string("static/html/ubcpi.html")
-
-        frag = Fragment(html)
+        '''html = ""
+        html += self.resource_string("static/html/ubcpi.html")'''
+
+        context.update(
+            {
+                'is_past_due': self.is_past_due(),
+                # 'options': self.options,
```

Comment:
This is a line I added and then commented (when it didn't work out) to try to resolve the parsing issue in the html.